web/v1/redshift/CHANGELOG

```diff
@@ -0,0 +1,12 @@
+Version 1.2.0 (2021-01-25)
+---------------------------
+Add config (#12)
+Update test for page_view_in_session_index (#13)
+Rename metadata tables and columns (#21)
+Rename variable to include unit of measurement (#23)
+Remove scan limit from engagement time (#5)
+Exclude NULL page_view_id rows from page view module (#20)
+Update inconsistent table definitions (#10)
+Remove second customisation example (#14)
+Update READMEs formatting (#7)
+Update copyright notices (#19)
```

Comment:
Could the "Add Changelog" commit be added too since it refers to issue (#6)?
Also just few minor differences from commit messages/issue titles.
e.g. `Update inconsistent table definitions` Vs `Redshift: Fix table definitions`

Reply:
Changelog should be added in the 'Prepare for release' commit - for no other reason than historically that's how we've done it. 

I just forgot to add one for the first release (but chengelog entry should exist for that release too).

We can have a 'add changelog' commit for review purposes and squash it with 'prepare for release' if we want.

Reply:
Ok that is fine. Do the commit names have to align with the issue exactly?

Reply:
Yes please - you can rename commits or rename issues to achieve it.